liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/diffChangelog.test.groovy

```diff
@@ -21,7 +21,7 @@ Long Description: NOT SET
 Required Args:
   changelogFile (String) Changelog file to write results
   referenceUrl (String) The JDBC reference database connection URL
-  url (String) The JDBC target database connection URL
+  url (String) The JDBC database connection URL
```

Comment:
Why is `url` defined as obfuscated but `referenceUrl` is not? Seems it should be both or neither.

Reply:
You are right. Fixing it